package.json

```diff
@@ -155,5 +156,11 @@
       "!src/test/*.js"
     ],
     "timers": "fake"
-  }
+  },
+  "bundlesize": [
+     {
+       "path": "dist/react.min.js",
```

Comment:
Did you verify this? I don't think a file with this name gets generated on master.
You probably want to add `build/dist/react.production.min.js` and `build/dist/react-dom.production.min.js` here.